src/common/services/contracts.js

```diff
@@ -0,0 +1,53 @@
+import RelayerContract from 'culturestake-contracts/build/contracts/Relayer.json';
+import AdminContract from 'culturestake-contracts/build/contracts/Admin.json';
+import VoteContract from 'culturestake-contracts/build/contracts/Vote.json';
+
+/**
+ * Helper method to get a deployed smart contract instance.
+ *
+ * @param {Web3} web3 - Web3 instance
+ * @param {Object} abi - contract abi
+ * @param {Object} address - contract address
+ *
+ * @return {Object} - contract instance
+ */
+function getContract(web3, abi, address) {
```

Comment:
We can get the `web` instance through an import here. It was not possible in the Circles implementation you copied the code from :smile_cat: